### PR TITLE
added fabform.io to FORMS

### PR DESCRIPTION
**Pull request by fabformG**
Added fabform.io to Forms section.

### CODE REVIEW

**Review by fabformG — COMMENTED** [review]
cleaned up the formatting, added the free tier in description.
Here is the pricing  https://fabform.io/pricing

cheers!

File: README.md [hunk 1]
```diff
@@ -834,6 +835,8 @@ Table of Contents
 
 ## Forms
 
+  * [FabForm].(https://fabform.io/) - Form backend platform for smart developers. Free plan allows 250 form submissions per month. Friendly modern GUI. Integrates with Google Sheets, Airtable, Slack, Email and others.
```
Comment (ripienaar) [inline README.md:838]: The format for links is `[description](url)` yours have a dot between. 

Also please remove the blank line. 

You click top of the PR and view the file to see how it renders.

**Review by fabformG — COMMENTED** [review]
fixed formatting.

**Comment by ripienaar** [comment]
Sorry this is just comical. Not spending more time on this one, if you can’t pay attention to what you are doing I am not going to waste my time.

**Pull request closed without merging**